lib/fathead/sqlalchemy/requirements.txt

```diff
@@ -0,0 +1 @@
+bs4
```

Comment:
I think this needs to declare a version?

You should use `pip freeze > requirements.txt` to generate the file